kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/log/StdoutLogger.kt

```diff
@@ -0,0 +1,120 @@
+package kotlinx.fuzz.log
+
+import java.time.LocalDateTime
+import org.slf4j.Logger
+import org.slf4j.Marker
+import org.slf4j.event.Level
+
+/**
+ * Basic slf4j logger implementation that works as a fallback when there are no slf4j service providers.
+ * By default, all the levels are enabled.
+ */
+@LoggerImpl
+internal object StdoutLogger : Logger {
+    private fun log(level: Level, msg: String?, t: Throwable? = null) {
+        val timestamp = LocalDateTime.now()
+        val threadName = Thread.currentThread().name
+        val message = "$timestamp [$threadName] [$level] - ${msg ?: ""}"
+        println(message)
+        t?.printStackTrace()
+    }
+
+    private fun isLevelEnabled(level: Level): Boolean = true
```

Comment:
This function always returns `true`. Is it intentional? Also, some of `isLevelEnabled` functions use this function (so they are always enabled) while other don't

Reply:
Yes, this is intentional. The core idea is that this logger implementation just outputs everything, and `LoggerWrapper` handles the actual level checks.

Other functions also should return true, I forgot to refactor them